lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -272,7 +272,7 @@ class DemographicsCollectionModal extends React.Component {
                     {gettext('Welcome to edX! Before you get started, please take a few minutes to fill-in the additional information below to help us understand a bit more about your background. You can always edit this information later in Account Settings.')}
                   </p>
                   <br />
-                  <span className="fa fa-info-circle" />
+                  <span aria-hidden className="fa fa-info-circle" />
```

Comment:
Do we not need to add `="True"` here?

Reply:
oh yeah, I thought react would just add it, guess not...